crash/app/src/main/java/com/google/samples/quickstart/crash/kotlin/MainActivity.kt

```diff
@@ -22,13 +22,14 @@ import com.google.samples.quickstart.crash.databinding.ActivityMainBinding
 class MainActivity : AppCompatActivity() {
 
     private lateinit var crashlytics: FirebaseCrashlytics
+    private lateinit var customKeySamples: CustomKeySamples
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         val binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        val customKeySamples = CustomKeySamples(applicationContext)
+        this.customKeySamples = CustomKeySamples(applicationContext)
```

Comment:
I believe `this` can be ommited?
```suggestion
        customKeySamples = CustomKeySamples(applicationContext)
```

Reply:
Definitely.